index.html

```diff
@@ -219,23 +200,26 @@ <h6>Skills</h6>
 </div>
 </section>
 
-</main>
 
-<div class="wrapper">
-  <footer class="footer">
-  
+<section id=Contact>
+<figure id="formimage"></figure>
+<div class="wrapper footer">
 <div><p id=footertext>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p></div>
```

Comment:
Great work on your contact section! I have a couple of suggestions that can help align it with the template and improve the overall presentation:

- [ ] Please check the `font size` of your paragraph in the contact section. It appears to be **slightly different from the template.** Adjusting the `font size` will ensure consistency and create a more harmonious visual experience. 🙋‍♂️

- [ ] Additionally, kindly review **the alignment of your paragraph in the contact section.** It seems to be misaligned compared to the template. Aligning it correctly **will contribute to a polished and professional look.** 🎯

   ### Your design:
   ![image](https://github.com/dvcorreia1984/portfolio/assets/107952724/53b4d808-7e02-4a0e-9a5f-af0e1c8238b9)


   ### Figma design:
   ![image](https://github.com/dvcorreia1984/portfolio/assets/107952724/630ee929-f9e9-40d0-b310-3fa24e10bb73)

Reply:
I also made these adjustments. Thanks for your comments.